src/app/pages/viewing/messages/messages.component.html

```diff
@@ -1,59 +1,84 @@
-<div class="row my-3">
-  <div class="col-12">
-    <div class="custom-control custom-switch">
-      <input type="checkbox" class="custom-control-input" id="customSwitch1" [(ngModel)]="viewingService.isCardView"
-             (click)="toggleViewStyle()">
-      <label class="custom-control-label toggle-switch" for="customSwitch1" *ngIf="!viewingService.isCardView">Switch to
-        Cards</label>
-      <label class="custom-control-label toggle-switch" for="customSwitch1" *ngIf="viewingService.isCardView">Switch to
-        Booklet</label>
+<!-- Perform a check to see if the number of messages received is 0 -->
+<div *ngIf="viewService.numMessages === 0; else showMessages" class="m-5" id="no-messages">No messages...
+  <markdown emoji>:sob:</markdown>
+</div>
+
+<!-- If the ngIf above fails, this ngTemplate will show -->
+<ng-template #showMessages>
+  <!-- Toggle switch for message viewing styles -->
+  <div class="row my-3">
+    <div class="col-12">
+      <div class="custom-control custom-switch">
+        <input type="checkbox" class="custom-control-input" id="customSwitch1" [(ngModel)]="viewService.isCarouselView"
+               (click)="toggleViewStyle()">
+        <label class="custom-control-label toggle-switch" for="customSwitch1" *ngIf="!viewService.isCarouselView">Switch to
+          Booklet</label>
```

Comment:
Line overflow (batch with previous comment)
```suggestion
          to Booklet</label>
```